plugins/modules/foreman_hostgroup.py

```diff
@@ -283,11 +284,6 @@ def main():
             parent=dict(type='entity', flat_name='parent_id'),
             organizations=dict(type='entity_list', flat_name='organization_ids'),
             locations=dict(type='entity_list', flat_name='location_ids'),
-            compute_resource=dict(type='entity', flat_name='compute_resource_id'),
```

Comment:
looking at the APIdoc, the following parameters are allowed for both, host and hostgroup:
```
environment_id
compute_profile_id
compute_resource_id
operatingsystem_id
architecture_id
pxe_loader
medium_id
ptable_id
subnet_id
subnet6_id
domain_id
realm_id
config_group_ids
puppet_proxy_id
puppet_ca_proxy_id
root_pass
```

(the other *_proxy_id probably too, I only looked at foremans apidoc)

so, shall we move all?

Reply:
Probably yes. I started with a small random set to see if it works.

Reply:
opened https://github.com/theforeman/foreman-ansible-modules/issues/647 to track this